### PR TITLE
[benchmark] Upgraded Benchmark test to support two quantized models

### PR DESCRIPTION
**Pull request by BruceDai**
@ibelem PTAL, thanks

### CODE REVIEW

**Comment by ibelem** [comment]
@Wenzhao-Xiang Do you have any comments?

**Comment by BruceDai** [comment]
@Wenzhao-Xiang @ibelem 
I'm adding another two missed quant models:
```
MobileNet v1 Quant (TFLite)
MobileNet v2 Quant (TFLite)
```
please review this PR later, thanks

**Comment by Wenzhao-Xiang** [comment]
@BruceDai  Thanks! Please also add `MobileNet v1 Quant (TFLite) / MobileNet v2 Quant (TFLite)` for Image Classification!

**Comment by BruceDai** [comment]
@Wenzhao-Xiang PTAL, thanks.

File: benchmark/index.html [hunk 1]
```diff
@@ -50,7 +50,9 @@ <h1>WebNN Benchmark</h1>
             <h4>Model</h4>
             <select id="modelName">
               <option value="mobilenet_v1_tflite" selected>Mobilenet v1 (TFLite)</option>
+              <option value="mobilenet_v1_quant_tflite" selected>Mobilenet v1 Quant (TFLite)</option>
```
Comment (Wenzhao-Xiang) [inline benchmark/index.html]: why `selected` here?

File: benchmark/index.html [hunk 2]
```diff
@@ -50,7 +50,9 @@ <h1>WebNN Benchmark</h1>
...
               <option value="mobilenet_v2_tflite">Mobilenet v2 (TFLite)</option>
+              <option value="mobilenet_v2_quant_tflite" selected>Mobilenet v2 Quant (TFLite)</option>
```
Comment (Wenzhao-Xiang) [inline benchmark/index.html]: ditto

File: benchmark/index.html (hunk 1 above)
Comment (BruceDai) [reply benchmark/index.html]: @Wenzhao-Xiang thanks, I have submitted another patch to fix this issue.

**Comment by Wenzhao-Xiang** [comment]
LGTM! @ibelem PTAL, thanks!

**Comment by ibelem** [comment]
LGTM, thanks!

**Pull request merged**